server/ngwmn/services/ngwmn.py

```diff
@@ -309,3 +310,179 @@ def get_features(latitude, longitude, service_root=SERVICE_ROOT):
+    """
+    Call ngwmn_cache for site statistics data.
+
+    :param agency_cd: string agency code
+    :param site_no: alphanumeric site number
+    :returns overall and monthly statistics
+
+    SAMPLE overall
+    {
+      "CALC_DATE": "2018-07-04",
+      "LATEST_VALUE": "213.84",
+      "MAX_DATE": "2018-04-10T13:18:00-06:00",
+      "AGENCY_CD": "USGS",
+      "MEDIAN_VALUE": "217.82",
+      "RECORD_YEARS": "31.6",
+      "MAX_VALUE": "181.7",
+      "MEDIATION": "BelowLand",
+      "MIN_VALUE": "232.96",
+      "SAMPLE_COUNT": "7456",
+      "SITE_NO": "353945105574502",
+      "MIN_DATE": "1986-09-26T12:00:00",
+      "LATEST_PCTILE": "0.78606",
+      "IS_RANKED": "Y"
+    }
+    SAMPLE monthly
+    {
+      "3": {
+        "RECORD_YEARS": "23",
+        "P50": "216.54",
+        "P25": "222.72",
+        "P10": "226.24",
+        "P50_MAX": "184.17",
+        "P90": "188.72",
+        "P75": "212.37",
+        "AGENCY_CD": "USGS",
+        "SAMPLE_COUNT": "673",
+        "MONTH": "3",
+        "SITE_NO": "353945105574502",
+        "P50_MIN": "227.46"
+      },
+      "1": {
+        "RECORD_YEARS": "20",
+        "P50": "218.28",
+        "P25": "225.50",
+        "P10": "227.03",
+        "P50_MAX": "184.14",
+        "P90": "190.49",
+        "P75": "214.81",
+        "AGENCY_CD": "USGS",
+        "SAMPLE_COUNT": "608",
+        "MONTH": "1",
+        "SITE_NO": "353945105574502",
+        "P50_MIN": "227.89"
+      }
+    }
+    """
+    site_info = monthly = {'IS_FETCHED': 'N'}
+
+    overall = get_statistic(agency_cd, site_no, 'wl-overall')
+    if overall['IS_FETCHED'] == 'Y':
+        site_info = get_statistic(agency_cd, site_no, 'site-info')
+        if overall['IS_RANKED'] == 'Y':
+            monthly = get_statistic(agency_cd, site_no, 'wl-monthly')
+
+    # handle to potential fetch fail with default
+    stats = {
+        'alt_datum': 'unknown',
+        'calc_date': 'unknown',
+        'overall': [],
+        'monthly': []
+    }
+
+    if overall['IS_FETCHED'] == 'Y':
+        alt_datum_cd = ''
+        if site_info['IS_FETCHED'] == 'Y':
+            alt_datum_cd = site_info['altDatumCd']
+
+        if overall['MEDIATION'] == 'BelowLand':
+            stats['alt_datum'] = 'Depth to water, feet below land surface'
+        else:
+            stats['alt_datum'] = 'Water level in feet relative to ' + alt_datum_cd
+
+        stats['calc_date'] = overall['CALC_DATE']
+
+        stats['overall'] = [
+            overall['MIN_VALUE'],
+            overall['MEDIAN_VALUE'],
+            overall['MAX_VALUE'],
+            overall['MIN_DATE'],
+            overall['MAX_DATE'],
+            overall['SAMPLE_COUNT'],
+            overall['RECORD_YEARS'],
+            overall['LATEST_VALUE'],
+            overall['LATEST_PCTILE']
+        ]
+
+    if monthly['IS_FETCHED'] == 'Y':
+        month_names = ["Non", "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+        for month in range(1, 12):
```

Comment:
So, this is range goes from 1 through 11. There's no way to every get `Non` or `Dec`. Is that intended behavior?